src/containers/BookForm.js

```diff
@@ -64,7 +64,7 @@ const BookForm = ({ createNewBook }) => {
             ))}
           </select>
         </label>
-        <input onClick={(e) => handleClick(e)} type="submit" value="New book" />
+        <input onClick={(e) => handleSubmit(e)} type="submit" value="New book" />
```

Comment:
According to the instructions on pathwright, you're supposed to attach the `handleSubmit` to the form opening tag just like following the best practice working with form, check out [form in React](https://reactjs.org/docs/forms.html).

![image](https://user-images.githubusercontent.com/31897434/124330375-2e5e5700-db85-11eb-99a2-e3ed4d09dd87.png)

Reply:
Hi!

It's actually attached in the form with onClick, Can you clarify this? Because the handleSubmit saves a new book on the Redux store and resets the component's state already.

![image](https://user-images.githubusercontent.com/60631456/124331497-ede6e500-db54-11eb-8e2a-2f3ffe81fb1c.png)